app/nginx-1.11.10/src/event/ngx_event.h

```diff
@@ -407,16 +413,59 @@ extern ngx_uint_t            ngx_use_epoll_rdhup;
 #define NGX_CLEAR_EVENT    0    /* dummy declaration */
 #endif
 
+#if (NGX_HAVE_FSTACK)
+
+    static inline ngx_int_t
```

Comment:
According to the orginal nginx, codes that wrapped with `#if` and `#endif` are no need to indent.

Reply:
I've fixed it.